contracts/SpokePool.sol

```diff
@@ -102,6 +102,8 @@ abstract contract SpokePool is
     // to eliminate any chance of collision between pre and post V3 relay hashes.
     mapping(bytes32 => uint256) public fillStatuses;
 
+    mapping(address => mapping(address => uint256)) public relayerRefund;
```

Comment:
Make sure you decrement the __gap array size at the bottom of the file to keep storage aligned.

Reply:
just decrement by 1 for a 2d mapping? I'm not totally sure how this gap variable works.

Reply:
Spoke implementations for each chain inherit this SpokePool, which is also upgradable, so we need to ensure the storage layout on derived contracts stays the same